.github/actions/verify-submodule-pull-request/Check-Parent.ps1

```diff
@@ -1,12 +1,13 @@
 param(
     [string] $Repository,
-    [string] $Title
+    [string] $Branch
 )
 
 $url = "https://api.github.com/repos/$Repository/pulls?state=open&per_page=100"
 $titles = curl -s -H 'Accept: application/vnd.github.v3+json' $url | ConvertFrom-Json | ForEach-Object { $_.title }
 
-$issueCode = $Title -replace '^\s*(\w+-\d+)\s*:.*$', '$1'
+$Branch -match '(\w+-\d+)'
+$issueCode = $matches[0]
```

Comment:
This failed for me here: https://github.com/Lombiq/Orchard-Chart.js/pull/47/checks. Any idea why?

Reply:
Is it a standardly formatted issue branch?

Reply:
Click the link, go, try!

![image](https://user-images.githubusercontent.com/164360/204056271-12f657c4-2c7f-452b-a79b-4f7c8cf9b6cc.png)